tests/app/main/test_frameworks.py

```diff
@@ -3993,7 +3993,8 @@ def test_we_abort_if_agreement_does_not_match_supplier_framework(
         supplier_framework = self.supplier_framework(framework_slug='g-cloud-8', on_framework=True)['frameworkInterest']
         return_supplier_framework.return_value = supplier_framework
 
-        res = self.client.get("/suppliers/frameworks/g-cloud-8/234/signer-details")
+        # Do request
```

Comment:
I'm kind of in 2 minds. 
The way I interpreted the intention here is that the original writer was assigning to `res` to make it clear that the code was getting a response at that point (which I actually quite like) :man_shrugging: